packages/client/src/contents/getSearchContents.ts

```diff
@@ -12,11 +24,14 @@ import type { Contents, QueryContents } from './types';
  * @param {string} query.spaceCode - The space the content belongs to (website|mobileapp|emailTool...).
  * @param {string} query.environmentCode - The environment identifier (live | preview).
  * @param {string} query.contentTypeCode - The content type unique code (page|post|menu|pages|posts|widgets|waterproof...).
- * @param {Array}  query.codes - List of codes that representing the content code (about-us|today-news|header|productId...).
- * @param {number} query.page - Number of the page to get, starting at 1. The default is 1.
- * @param {number} query.pageSize - Size of each page, as a number between 1 and 180. The default is 60.
- * @param {object} query.target - The targets and respective values that a content type is configured (contentzone:ROW | country:GB | language:en-GB | benefits:test).
+ * @param {string|string[]} [query.codes] - List of codes that representing the content code (about-us|today-news|header|productId...).
+ * @param {QueryTarget} [query.target] - The targets and respective values that a content type is configured (contentzone:ROW | country:GB | language:en-GB | benefits:test).
```

Comment:
Maybe the example should be `{contentzone: ROW}` and the other examples, included in QueryTarget type definition above.